packages/components/src/standalone-comparison-grid/index.tsx

```diff
@@ -0,0 +1,68 @@
+import { Button } from '@wordpress/components';
+import './style.scss';
+
+interface StandAloneComparisonGridColumn {
+	title: string;
+	rows: string[];
+	intro_copy?: string;
+	action_copy: string;
+	action: () => void;
+}
+
+export interface StandAloneComparisonGridProps {
+	columns: StandAloneComparisonGridColumn[];
+}
+
+const StandAloneComparisonGrid = ( props: StandAloneComparisonGridProps ) => {
+	const { columns } = props;
+
+	return (
+		<div className="standalone-comparison-grid">
+			{ columns.map( ( column, index ) => {
+				return (
+					<div className="standalone-comparison-grid__column" key={ `scd_${ index }` }>
+						<table className="standalone-comparison-grid__body">
```

Comment:
something tells me that we shouldn't be using tables for layout. Did you think about using grid-templates-columns + flex?
Is there a specific reason you prefer tables?

Reply:
I went with tables because I was trying to steal the styles from the existing plans-grid and that's what it uses.

But this is probably a good opportunity to _not_ use tables and more modern stuff instead!

Reply:
Yes, we can easily use a regular list and two containers with a flex box to solve it.